provers/scripts/parameterize-zkps.ts

```diff
@@ -0,0 +1,35 @@
+#! ./node_modules/.bin/ts-node
+
+import 'zx/globals';
+
+(async (): Promise<void> => {
+  let paramsAll: any[] = [];
+
+  // generate parameters for the auto-generated verifier contract
+  fs.readdir('./circuit-inputs/', async (_, filenames) => {
+    for (let i = 0; i < filenames.length - 1; i++) {
+      if (filenames[i] !== '.gitkeep') {
+        echo(`\nGenerating parameters of proof-${i}.json for verifier contract...`);
+        // NOTE: renaming file names here because `snarkjs generatecall` only accepts certain file names
```

Comment:
This is because `snarkjs` hardcoded file names in their source [code](https://github.com/iden3/snarkjs/blob/master/build/cli.cjs).